lib/actions/deleteLinkFromProjectAction.js

```diff
@@ -0,0 +1,32 @@
+const assert = require('assert');
+const DeleteLinkFromProjectResponse = require('./deleteLinkFromProjectResponse');
+const ProjectLink = require('../projectLink');
+
+class DeleteLinkFromProjectAction {
+  constructor(projectLink = ProjectLink) {
+    this.projectLink = projectLink;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  parseParams(userInputString = '') {
+    assert(typeof userInputString === 'string');
+    const parsedUserInput = userInputString.split(' ');
+
+    if (parsedUserInput.length < 2) {
+      throw new TypeError('invalid input');
+    }
+    const linkParams = { label: parsedUserInput[1], link: parsedUserInput[2] };
```

Comment:
We will only need the label of the link and not the URL, in order to delete. So we would not expect nor need a link parameter, right?